server/views/macros/remandCard.njk

```diff
@@ -1,6 +1,5 @@
 {% macro remandCard(remand, model, relevant) %}
-    {% set colour = 'red' if not relevant else ('green' if model.isApplicable(remand) else 'blue')%}
-    {% set text = 'Not applicable' if not relevant else ('Applicable' if model.isApplicable(remand) else 'Shared')%}
+    {% set colour = 'red' if not relevant else ('green' if remand.status == 'APPLICABLE' else 'blue')%}
```

Comment:
ESLint on is very against nested ternaries. Should the same rule apply to `.njk` files?

Reply:
Yea probably. I'll lookat fixing it in the next PR.